monai/networks/trt_compiler.py

```diff
@@ -180,7 +184,8 @@ def try_set_inputs():
                     raise
                 self.cur_profile = next_profile
                 ctx.set_optimization_profile_async(self.cur_profile, stream)
-
+            except Exception:
+                raise
```

Comment:
Could you please add more info to explain this exception?
Thanks,  
Bin

Reply:
This would be exception trying to set input shapes for which the engine was not built ; previously I had a logic there that would try rotating trt optimization profile index on such an exception - we do not use multiple profiles with MONAI so I should probably simplify the code.

Reply:
Actually, let's just leave the profile selection code here, it does not hurt.